docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,144 @@
+
+  (!docs/pages/includes/add-role-to-user.mdx role="demo-access-request" user="\`demo-access-requester\`"!)
+
+## Step 2/4. Assign user traits
+
+To allow automatic review rules to evaluate the requesting user, assign them
+traits via the Teleport Web UI.
+
+1. Go to **Zero Trust Access** -> **Users**
+2. Next to `demo-access-requester`, click **Options** -> **Edit...**
+3. Click **Add user trait**, and set:
+   - Key: `team`
+   - Value: `demo`
+4. Click **Save**
+5. Verify that the user has been updated with the desired trait.
+
+![Edit User Traits](../../../../img/access-controls/automatic-reviews/users-edit.png)
+
+When adding user traits, you can enter any keys and values. The user trait form
+does not support wildcard or regular expressions.
+
+## Step 3/4. Create Access Monitoring Rule
+
+Next, define the automatic review rule via the Teleport Web UI.
+
+1. Go to **Identity Governance** -> **Access Requests** -> **View Access Monitoring Rules**
+2. Click **Create Access Monitoring Rule** -> **Automatic Review Rule**
+3. Configure the rule and set:
+   - **Name of roles to match**: `access`
+   - **User Traits**: `team: demo`
+   - **Review decision**: `APPROVED`
+4. Click **Create Rule**
+
+![Create Automatic Review Rule Editor](../../../../img/access-controls/automatic-reviews/create-automatic-review-rule.png)
+
+This Access Monitoring Rule will ensure that Access Requests for the `access`
+role will be automatically reviewed for approval if the Teleport user traits
+requirements are satisfied. In this case, any user with the traits `team: demo`
+will satisfy the requirement.
+
+## Step 4/4. Verify automatic review rule
+
+To verify the new Access Monitoring Rule, create an Access Request via the Teleport
+Web UI.
+1. Log in as `demo-access-requester`
+2. Go to **Access Requests** and click **New Access Request**
+3. Change the request type from **Resources** to **Roles**
+4. Add the `access` role to the Access Request
+5. Click **Proceed to Request**, then **Submit Request**
+
+![Create Access Request](../../../../img/access-controls/automatic-reviews/create-access-request.png)
+
+At this point, the new Access Request should have been created, automatically
+reviewed, and transitioned into an `APPROVED` state. Navigate **Back to Listings**
+and verify the Access Request status. It might take a second for the review to
+process, so you may have to refresh the page.
+
+![View Access Requests](../../../../img/access-controls/automatic-reviews/view-access-requests.png)
+
+## Troubleshooting
+
+### Conflicting automatic review rules
+
+Automatic review rules can automatically approve or deny Access Requests based
+on the selected review decision. If an Access Request meets the conditions for
+both an approval rule and a denial rule, neither rule will be applied.
+
+If you suspect this issue, review your Access Monitoring Rules in the Teleport
+Web UI or by running `tctl get access_monitoring_rules`.
```

Comment:
A few comments.

1. It's an interesting decision to apply neither. Typically in Teleport's RBAC system when multiple rules disagree then the strictest option wins. Any particular reason we made automatic reviews work differently?

2. From a UX perspective, "manually review all of your access monitoring rules to see if there's a conflict" is not very friendly. Can we detect this case and communicate the conflict to the user? (Not saying we need to do it now, but maybe we can create an issue to follow up).

Reply:
> Typically in Teleport's RBAC system when multiple rules disagree then the strictest option wins.

That probably makes more sense. I'll create a follow up item make this change.

> From a UX perspective, "manually review all of your access monitoring rules to see if there's a conflict" is not very friendly.

Yeah, this is a concern that I've been thinking about how to address. We don't have a great solution for validating or reviewing access_monitoring_rules right now. The use of Teleport's predicate language makes this problem a bit complicated.

Reply:
https://github.com/gravitational/teleport/pull/54933 changes the behavior when conflicting rules match a request. Denied reviews will now take precedence.